docs/content/doc/installation/with-docker.en-us.md

```diff
@@ -331,25 +343,26 @@ The idea of this option is to use (essentially unchanged) the authorized_keys th
     - USER_GID=1000
   ```
 
-- Mount `/home/git/.ssh` of the host into the container. Otherwise the SSH authentication cannot work inside the container.
+- Mount `/home/git/.ssh` of the host into the container. This ensures that the `authorized_keys` file is shared between the host `git` user and the container `git` user. Otherwise the SSH authentication cannot work inside the container.
```

Comment:
```suggestion
- Mount `/home/git/.ssh` of the host into the container. This ensures that the `authorized_keys` file is shared between the host `git` user and the container `git` user otherwise the SSH authentication cannot work inside the container.
```